.github/workflows/actions.yaml

```diff
@@ -25,3 +30,79 @@ jobs:
         run: |
           cd test
           uv run -p ${{ matrix.python-version }} --resolution ${{ matrix.resolution }} pytest
+
+  # Test commands in a dummy environment
+  commands:
+    runs-on: ubuntu-22.04 # this is the lastest release supported by MongoDB
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
+        resolution: ["highest", "lowest-direct"]
+    # MySQL for Slurm
+    services:
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - "8888:3306"
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.11.0
+        with:
+          mongodb-username: admin
+          mongodb-password: admin
+          mongodb-port: 27017
+      - name: Install mongosh
+        run: |
+          sudo apt-get install gnupg
+          wget -qO- https://www.mongodb.org/static/pgp/server-7.0.asc | sudo tee /etc/apt/trusted.gpg.d/server-7.0.asc
+          echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+          sudo apt-get update
+          sudo apt-get install -y mongodb-mongosh
+          mongosh --version
+      - name: Create seml MongoDB user
+        run: |
+          mongosh --host localhost:27017 -u admin -p admin --authenticationDatabase admin <<EOF
+            db = db.getSiblingDB('seml');
+            db.createUser({ user: 'seml', pwd: 'seml', roles: [{ role: 'readWrite', db: 'seml' }] });
+          EOF
+      - name: Setup Slurm Cluster
+        uses: koesterlab/setup-slurm-action@v1
+      # Checkout the code
+      - uses: actions/checkout@v4
+      - name: Install uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
+      - name: Setup virtual env
+        run: uv sync
+      - name: Setup seml config
+        run: uv run seml configure --host=localhost --port=27017 --database=seml --username=seml --password=seml
+      - name: Add
+        run: uv run seml test_collection add ci/examples/example_config.yaml
+      - name: Start
+        run: uv run seml test_collection start
+      - name: Hold
```

Comment:
Maybe add a short wait? Potentially also in the experiment script? It may be that experiments finish fast / slow and this can affect this behaviour, can't it?

Reply:
since we currently only test whether seml throws an exit code which is not 0 (or an exception). I wouldn't really matter here. We have no checks yet that the commands do what they should. It only guarantees that seml is not crashing.

Reply:
But this may also depend on the state of the slurm queue... right?